cvmfs/cvmfs.cc

```diff
@@ -1410,6 +1410,12 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     attribute_value = string(VERSION) + "." + string(CVMFS_PATCH_LEVEL);
   } else if (attr == "user.pubkeys") {
     attribute_value = mount_point_->signature_mgr()->GetActivePubkeys();
+  } else if (attr == "user.catalog_counters") {
+    std::string subcat_path;
+    catalog::Counters counters =
```

Comment:
That needs to go up in the fenced region (before line 1398). Otherwise the path could become invalid in the meantime.